AutoCook/media/lua/client/AutoCook.lua

```diff
@@ -396,3 +495,6 @@ function AutoCook:new(playerObj, recipe, baseItem)
     
     return o;
 end
+
+-- load persisted values on reload script
+AutoCook:init(getPlayer())
```

Comment:
Keeping active debug code is only ok within a "false" condition by default. Maybe add some AutoCook.Debug same as there is an AutoCook.Verbose

Reply:
agreed, will only happen in debug mode now